include/wx/osx/cocoa/stdpaths.h

```diff
@@ -35,7 +35,7 @@ class WXDLLIMPEXP_BASE wxStandardPaths : public wxStandardPathsBase
     virtual wxString MakeConfigFileName(const wxString& basename,
                                         ConfigFileConv conv = ConfigFileConv_Ext
                                         ) const override;
-
+    virtual wxString GetSharedLibrariesDir() const override;
```

Comment:
For readability, don't replace the empty line, just add a new line.
Also in other places.